NOTICE

```diff
@@ -10,3 +10,7 @@ License. The source code of jersey can be found at https://github.com/jersey/jer
 This distribution has a binary test dependency on jqwik, which is available under
 the Eclipse Public License 2.0. The source code can be found at
 https://github.com/jlink/jqwik.
+
+The streams-scala (streams/streams-scala) module was donated by Lightbend and the original code was copyrighted by them:
+Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+Copyright (C) 2017-2018 Alexis Seigneurin.
```

Comment:
I'm not totally comfortable just dropping those notices, so I added this note here. Not sure if it's any better, though.

Reply:
I agree. We should preserve attribution if we can. "Community first".